gogio/iosbuild.go

```diff
@@ -164,31 +166,53 @@ func exeIOS(tmpDir, target, app string, bi *buildInfo) error {
 		return err
 	}
 	mainm := filepath.Join(tmpDir, "main.m")
-	const mainmSrc = `@import UIKit;
+	mainmSrc := struct {
+		Deeplink []string
+	}{
+		Deeplink: bi.deeplink,
+	}
+
+	mainTmpl, err := template.New("").Parse(`@import UIKit;
 @import Gio;
 
 @interface GioAppDelegate : UIResponder <UIApplicationDelegate>
 @property (strong, nonatomic) UIWindow *window;
+@property (strong, nonatomic) GioViewController *controller;
```

Comment:
Can the `controller` field be replaced with `window.rootViewController`, cast to a `GioViewController`?

Reply:
It's not possible to a external code modify `window.rootViewController` (and then, not be a `GioViewController` anymore)?

Reply:
I'd say as long as we own the app delegate, it is an error to replace `self.window.rootViewController`. On the other hand, a mis-cast of a replaced controller is probably hard to diagnose. Let's keep your change.